.github/workflows/runTests.yaml

```diff
@@ -55,7 +55,9 @@ jobs:
     - name: Setup Environment
       uses: ./.github/actions/setupEnvironment
     - name: Setup database
-      run: psql postgres://postgres:password@localhost:5433/postgres -f schema/accepted_schema.sql
+      run: |
+        psql postgres://postgres:password@localhost:5433/postgres -f schema/accepted_schema.sql
+        psql postgres://postgres:password@localhost:5433/postgres -f playwright/fixtures.sql
```

Comment:
We had to add fixtures to add a database setting with a crosspost signing key. We're not actually using the fixtures in the "normal" playwright tests, but I added them here anyway otherwise it'll be confusing if we want to do so in the future.